ext/RMagick/rmutil.c

```diff
@@ -1445,6 +1445,7 @@ rm_progress_monitor(
     const MagickSizeType sp,
     void *client_data)
 {
+#if !defined(_WIN32)
```

Comment:
Can you explain what this is for?

Reply:
The SEGV has been occurred with old ImageMagick (v6.8.9) on only Windows.

If you set the `Proc` object through the `#monitor=` method, the object will be called each time ImageMagick processing progresses at https://github.com/rmagick/rmagick/blob/b1828e24af5eeb1e2b3c2f3e592552950a27bb6b/ext/RMagick/rmutil.c#L1464.

The Proc object is called correctly only once and SEGV always occurs in the second call.

I guess that there is a problem in callback processing with old ImageMagick and it breaks Proc object or something...